docs/chapters/sdk/installing-the-sdk.rst

```diff
@@ -0,0 +1,29 @@
+Downloading and installing the SDK
+==================================
+
+Download the SDK for your hardware from the `PELUX website`_, it is quite big because it contains tho whole sysroot for the project in addition to some native tools, so depending on your internet connectivity it could take a couple of minutes.
+
+If you download it from a HTTP server then the file per default is not executable, we need to change it with `chmod`. After that is done we can run it, it's a self extracting shell script.
+
+.. code-block:: bash
+                
+    cd ~/Downloads
+    chmod +x poky-glibc-x86_64-core-image-pelux-qt-corei7-64-toolchain-2.2.1.sh
+    ./poky-glibc-x86_64-core-image-pelux-qt-corei7-64-toolchain-2.2.1.sh
```

Comment:
When I started with Yocto I found the SDK version quite confusing as it wasn't incremented on every build but it indicated the Yocto version, `2.2.1` is in fact the version of Poky, in this case Morty, that is being used.